mvtprovider/postgis/README.md

```diff
@@ -0,0 +1,127 @@
+# PostGIS
+The PostGIS provider manages querying for tile requests against a Postgres database (version 12+) with the [PostGIS](http://postgis.net/)(version 3.0+) extension installed. The connection between tegola and Postgis is configured in a `tegola.toml` file. An example minimum connection config:
+
+
+```toml
+[[mvt_providers]]
+name = "test_postgis"       # provider name is referenced from map layers (required)
+type = "postgis"            # the type of data provider must be "postgis" for this data provider (required)
+host = "localhost"          # PostGIS database host (required)
+port = 5432                 # PostGIS database port (required)
+database = "tegola"         # PostGIS database name (required)
+user = "tegola"             # PostGIS database user (required)
+password = ""               # PostGIS database password (required)
+```
+
+### Connection Properties
+
+- `name` (string): [Required] provider name is referenced from map layers. please note that when referencing an mvt_provider form a map layer the provider name must be prexied with `mvt_`. See example config below.
+- `type` (string): [Required] the type of data provider. must be "postgis" to use this data provider
+- `host` (string): [Required] PostGIS database host
+- `port` (int): [Required] PostGIS database port (required)
+- `database` (string): [Required] PostGIS database name
+- `user` (string): [Required] PostGIS database user
+- `password` (string): [Required] PostGIS database password
+- `srid` (int): [Optional] The default SRID for the provider. Defaults to WebMercator (3857) but also supports WGS84 (4326)
+- `max_connections` (int): [Optional] The max connections to maintain in the connection pool. Defaults to 100. 0 means no max.
+
+## Provider Layers
+In addition to the connection configuration above, Provider Layers need to be configured. A Provider Layer tells tegola how to query PostGIS for a certain layer. An example minimum config:
+
+```toml
+[[mvt_providers.layers]]
+name = "landuse"
+# this table uses "geom" for the geometry_fieldname and "gid" for the id_fieldname so they don't need to be configured
+sql = "SELECT ST_AsMVTGeom(geom,!BBOX!) AS geom, gid FROM gis.landuse WHERE geom && !BBOX!"
+```
+
+### Provider Layers Properties
+
+- `name` (string): [Required] the name of the layer. This is used to reference this layer from map layers.
+- `geometry_fieldname` (string): [Optional] the name of the filed which contains the geometry for the feature. defaults to `geom`.
+- `id_fieldname` (string): [Optional] the name of the feature id field. defaults to `gid`.
+- `srid` (int): [Optional] the SRID of the layer. Supports `3857` (WebMercator) or `4326` (WGS84).
```

Comment:
Does this apply? For MVTProviders we have no control over reprojecting the data (for now)